index.md

```diff
@@ -138,12 +138,12 @@ The eventual goal is to let people stop worrying about administering machines an
 
 <a href="/images/gadfly-demo.png" target="_blank"><img src="/images/gadfly-demo.png" width="100%" /></a>
 
-# Free, Open Source and Library-Friendly
+# Free, Open Source and library-friendly
```

Comment:
The capitals are here because the heading is using title case, just like in other places. Else, "open source" would have been in small caps too (https://en.wikipedia.org/wiki/Open_source).